edward/inferences/inference.py

```diff
@@ -169,11 +171,25 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
     logdir : str, optional
       Directory where event file will be written. For details,
       see ``tf.summary.FileWriter``. Default is to write nothing.
+    logrun : str, optional
```

Comment:
Users can already separate info each time they refit a model by tweaking the `logdir` argument.  A typical use case would be
```python
inference.initialize(logdir='log/lr_{}_nsamples_{}'.format(learning_rate_stepsize, n_samples))
```
With `logdir`, users have full flexibility in setting however many subdirectories and filenames they want. `logdir + logrun` sort of imposes structure, where a user can specify `logrun` already inside `logdir`.

That said, I love the idea of having a default subdirectory inside `logdir` which timestamps the run. It seems impossible to turn it on/off without having an additional argument. So then maybe `logrun` should just be a boolean, and its name+default be changed to something like `log_timestamp=True`?